sickbeard/webserve.py

```diff
@@ -3162,7 +3172,7 @@ def manageTorrents(self):
             else:
                 t.webui_url = re.sub('localhost', sickbeard.LOCALHOST_IP, sickbeard.TORRENT_HOST)
         else:
-            t.webui_url = sickbeard.TORRENT_HOST
+            t.webui_url = sickbeard.TORRENT_HOST + 'web/'
```

Comment:
Ooops that change wasn't supposed to be included, it's specific to my transmission installation. Have to fix this, sorry for the mistake

<div>-------- Message d'origine --------</div><div>De : Fernando <notifications@github.com> </div><div>Date :06/01/2015  17:05  (GMT+01:00) </div><div>A : SiCKRAGETV/SickRage <SickRage@noreply.github.com> </div><div>Cc : CoRpO <corpo@gruk.org> </div><div>Objet : Re: [SickRage] Fix for sickragetv/sickrage-issues#347 (#1172) </div><div>
</div>In sickbeard/webserve.py:

> @@ -3162,7 +3172,7 @@ def manageTorrents(self):
>              else:
>                  t.webui_url = re.sub('localhost', sickbeard.LOCALHOST_IP, sickbeard.TORRENT_HOST)
>          else:
> -            t.webui_url = sickbeard.TORRENT_HOST
> +            t.webui_url = sickbeard.TORRENT_HOST + 'web/'
@CoRpO why did you change this? 
this will break transmission.
I think you should add another conditional clause.

—
Reply to this email directly or view it on GitHub.